src/core/agent/services/multiSigService.ts

```diff
@@ -990,38 +1017,70 @@ class MultiSigService extends AgentService {
       .identifiers()
       .get(ourIdentifier.signifyName);
 
-    const time = timeAdmit
-      ? timeAdmit
-      : new Date().toISOString().replace("Z", "000+00:00");
-
-    const [admit, sigs, end] = await this.props.signifyClient
-      .ipex()
-      .admit(multisigSignifyName, "", grantSaid, time);
-
-    const mstate = gHab["state"];
-    const seal = [
-      "SealEvent",
-      { i: gHab["prefix"], s: mstate["ee"]["s"], d: mstate["ee"]["d"] },
-    ];
-    const sigers = sigs.map((sig: any) => new Siger({ qb64: sig }));
-    const ims = d(messagize(admit, sigers, seal));
-    let atc = ims.substring(admit.size);
-    atc += end;
-    const gembeds = {
-      exn: [admit, atc],
-    };
     const recp = multisigMembers
       .filter((signing: any) => signing.aid !== ourIdentifier.id)
       .map((member: any) => member.aid);
 
-    const [exn, sigsMes, dtime] = await this.props.signifyClient
-      .exchanges()
-      .createExchangeMessage(
-        mHab,
-        "/multisig/exn",
-        { gid: gHab["prefix"] },
-        gembeds
-      );
+    if (multisigExn) {
+      const [, ked] = Saider.saidify(multisigExn);
+      const admit = new Serder(ked);
+
+      const keeper = await this.props.signifyClient.manager!.get(gHab);
+      const sigs = await keeper.sign(b(JSON.stringify(multisigExn)));
```

Comment:
hmm, are we sure this shouldn't be `b(new Serder(multisigExn).raw)` or something like that?